docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,94 @@
 ---
-title: Preparing a Site to Go Live
+title: Eine Seite für die Veröffentlichung vorbereiten
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Wow! You've come a long way! You've learned how to:
+Wow! Du hast es weit geschafft. Du hast gelernt:
 
-- create new Gatsby sites
-- create pages and components
-- style components
-- add plugins to a site
-- source & transform data
-- use GraphQL to query data for pages
-- programmatically create pages from your data
+- eine neue Gatsby Seite zu erstellen
+- Pages und Components zu erstellen
+- Components zu stylen
+- Plugins zu installieren
+- Datenquellen zu erstellen und zu transformieren 
+- GraphQL zu nutzen, um Daten abzufragen
+- programmatisch Pages aus deinen Daten zu erstellen 
 
-In this final section, you're going to walk through some common steps for preparing a site to go live by introducing a powerful site diagnostic tool called [Lighthouse](https://developers.google.com/web/tools/lighthouse/). Along the way, we'll introduce a few more plugins you'll often want to use in your Gatsby sites.
+In diesem letzten Abschnitt wirst du die gängigen Schritte durchlaufen, um deine Seite für die Veröffentlichung vorzubereiten. Dazu werden wir das mächtige Diagnose-Tool [Lighthouse](https://developers.google.com/web/tools/lighthouse/) benutzen. Außerdem werden wir noch einige weitere, häufig genutzte Plugins installieren.
 
-## Audit with Lighthouse
+## Audit mit Lighthouse
```

Comment:
```suggestion
## Auditieren mit Lighthouse
```

Reply:
That sounds a little awkward/unnatural imho, don't you think? How about something like `Überprüfung durch ein Lighthouse Audit` ?

Reply:
`Überprüfung durch einen Lighthouse Audit` sounds lovely!

Reply:
Cool! I commited the change. I also changed `zu Nütze` to `zu Nutze`!